tools/macos/tweak-dockermachine.sh

```diff
@@ -9,6 +9,7 @@ MACHINE_NAME=${1:-whisk}
 # Disable TLS.
 docker-machine ssh $MACHINE_NAME "echo DOCKER_TLS=no |sudo tee -a /var/lib/boot2docker/profile > /dev/null"
 docker-machine ssh $MACHINE_NAME "echo DOCKER_HOST=\'-H tcp://0.0.0.0:4243\' |sudo tee -a /var/lib/boot2docker/profile > /dev/null"
+docker-machine ssh $MACHINE_NAME "echo EXTRA_ARGS=\'--userns-remap=default\' |sudo tee -a /var/lib/boot2docker/profile > /dev/null"
```

Comment:
when docker-machine launches the daemon it reads the boot2docker/profile and appends EXTRA_ARGS to the dockerd parameters. there is no special env var for the userns flag. check `/usr/local/etc/init.d/docker` on a docker-machine VM.